pages/static/sitemap/sitemap_ampbyexample.xml

```diff
@@ -185,8 +185,9 @@
 <url> <loc>https://ampbyexample.com/stories/user_consent/story_user_consent/</loc> </url>
 <url> <loc>https://ampbyexample.com/stories/visual_effects/ken_burns/</loc> </url>
 <url> <loc>https://ampbyexample.com/user_consent/advanced_user_consent_flow/</loc> </url>
-<url> <loc>https://ampbyexample.com/user_consent/basic_user_consent_flow/</loc> </url>
+<url> <loc>https://ampbyexample.com/user_consent/client_side_user_consent_flow/</loc> </url>
```

Comment:
you don't need to fix the URL here, instead setup a redirect in here: https://github.com/ampproject/amp.dev/blob/future/platform/config/amp-dev-redirects.yaml#L38